wasmbinding/bindings/msg.go

```diff
@@ -0,0 +1,43 @@
+package bindings
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type SeiMsg struct {
```

Comment:
where do we use this struct? I saw that we use the other structs for encoding from the wasm struct to the proto generated struct for the message, but not sure why we need this one? @Kbhat1

Reply:
Yea sorry this should be removed